src/quo/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -24,6 +24,9 @@
           :on-press                (fn []
                                      (when on-press
                                        (on-press label)))
+          :on-long-press           (fn []
+                                     (when (fn? on-long-press)
+                                       (on-long-press label)))
```

Comment:
Maybe just 
```
#(when (fn? on-long-press)
    (on-long-press label))
```

Reply:
@briansztamfater 
Previously I'd suggest the same, but some time ago @ilmotta  suggested to only use `#` for simple one-liner functions.

The reason is it takes longer for the reader to realize if `%`, (or `%1`,  `%2`, etc...) is being used and where is placed.  Instead if we have `(fn[])` we know what the function takes.